build-logic/build-parameters/build.gradle.kts

```
@@ -0,0 +1,35 @@
+plugins {
+	id("org.gradlex.build-parameters") version "1.4.1"
+}
+
+group = "org.junit.gradle"
+
+buildParameters {
+	bool("ci") {
+		description.set("Whether or not this build is running a CI environment")
+		defaultValue.set(false)
+		fromEnvironment()
+	}
+	group("buildCache") {
+		string("username") {
+			description.set("Username to authenticate with the remote build cache")
+			fromEnvironment()
+		}
+		string("Password") {
```

❔ Does lower/upper case matter here?

---

Yes, it is. This should be lowercase.
```suggestion
		string("password") {
```